### ISSUE TITLE
accessing view options and image widget from 'modules'

### ISSUE THREAD

**Issue by mauricefallon** [RobotLocomotion/director#404]
@patmarion set up  "modules" to allow customising the director by calling an additional python script on startup. It allows importing robotSystem which gives access ikPlanner, jointController and even the view class.

But it seems I cannot access "view options" without doing om.findObjectbyName and I cannot interact with the imageWidget class (e.g. imageWidget.setWidgetSize(300))

Is is possible to access more of the UI configuration from a "module"?

**Comment by patmarion** [comment]
Yes, you should be able to access every variable that was declared in director's startup.py.  I assume you are running a command line:

    director --startup mymodule/runstartup.py

In the script runstartup.py, we have these lines:


    import mymodule.startup
    mymodule.startup.startup(robotSystem, globals())

So there are two args there 1) robotSystem 2) a globals dictionary.  The globals dictionary has all the variables defined in director's startup.py.  You can access them in your module like this:

    viewOptions = globalsDict['viewOptions']
    imageWidget = globalsDict['imageWidget']

If you want to formalize a set of variables into a data structure to pass around, you can do something like this:

    from director.fieldcontainer import FieldContainer
    variablesToExport = ['viewOptions', 'imageWidget']

    uiElements = FieldContainer(**{name:globalsDict[name] for name in variablesToExport})

    # then for example...
    uiElements.viewOptions.setProperty('View angle', 60)


In the above code, `uiElements` is like `robotSystem` but contains variable's from director's startup.py.  If you want to standardize the set of variables we export from director's startup.py then we could even define this structure in director.  Something like that has been on my todo list.
